docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.7'
+
+services:
+  db:
+    image: mongo:latest
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: rootpassword
+      MONGO_INITDB_DATABASE: catchall
+    ports:
+      - 27017:27017
+    volumes:
+      - mongodb_data_container:/data/db
+  
+  api:
+    build: ./
+    command: gin --appPort=9090 -i serve-http
```

Comment:
using gin on prod is not gonna give you the highest performance but you just just mention it's for testing purpose